### PR TITLE
Changing to make CustomFunctionMappings required

### PR DESCRIPTION
**Pull request by mscharlock**
Also got rid of s on Function

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mscharlock-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809251758257890-413%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MjnYFt3M4yYM34QjnVRVomA7vC9a%2B8KjjUfBIHge1o8%3D&st=2018-09-25T17%3A54%3A37Z&se=2018-10-26T17%3A59%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ If you are testing your add-in in Office 2016 desktop, you can enable [runtime l
...
+By default, the name of a custom function in your JavaScript file is typically declared using entirely uppercase letters, and corresponds exactly to the function name that end users see in Excel. However, you can change this by using the `CustomFunctionMappings` object to map one or more function names from the JavaScript file to different values that end users will see as function names in Excel. This is helpful if you are using an uglifier, webpack, or import syntax - all of which have difficulty with uppercase function names. `CustomFunctionMappings` required in both JavaScript and TypeScript projects.
```
Comment (kbrandl) [inline docs/excel/custom-functions-best-practices.md]: This final sentence is missing the word **is** (before "required in both....").

**Comment by mscharlock** [comment]
I agree, it sounds like this thing is different than what was first explained to me. Changed the is!

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/mscharlock-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809251811534434-413%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=aCeQIEB%2FZVm036S8T3Ymaax4wb03crN7jvB%2FN2z76uQ%3D&st=2018-09-25T18%3A08%3A05Z&se=2018-10-26T18%3A13%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**